tests/generic_config_updater/test_eth_interface.py

```diff
@@ -220,7 +220,7 @@ def test_replace_fec(duthost, ensure_dut_readiness, fec):
 
 
 @pytest.mark.parametrize("index, is_valid", [
-    ("33", True),
+    ("29", True),
```

Comment:
@isabelmsft why 29? NOT 30, 31, 32 based upon max number of ports on a platform?

Reply:
I chose 29 because it should work for any platform. Is there some benefit to changing this value to be variable based on max number of ports on a platform? 

In other GCU E2E tests, we typically chose any value that would work.